docs/content/preview/releases/yba-releases/v2024.2.md

```diff
@@ -76,7 +76,7 @@ For more information, refer to [Automatically provision on-premises nodes](/stab
 * [Backup portability](/stable/yugabyte-platform/back-up-restore-universes/restore-universe-data/#advanced-restore-procedure). After sending backups to a cloud object store (such as S3, GCS, or Azure blob storage), you can now copy those backup sets to other cloud storage buckets and restore from them.
 
 * Kubernetes
-  * [Kubernetes Resource Specification](/stable/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes/#instance-configuration). You can now configure CPU and memory needs for YB-TServer and YB-Master pods natively, instead of using the instance types API.
+  * [Kubernetes resource Specification](/stable/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes/#instance-configuration). You can now configure CPU and memory needs for YB-TServer and YB-Master pods natively, instead of using the instance types API.
```

Comment:
```suggestion
  * [Kubernetes resource specification](/stable/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes/#instance-configuration). You can now configure CPU and memory needs for YB-TServer and YB-Master pods natively, instead of using the instance types API.
```